### PR TITLE
Adding missing functions in SubKey class type definition

### PR DESCRIPTION
**Pull request by FabienForestier**
Typings of the SubKey class is still incomplete but I added a few of the missing function type definitions.

### CODE REVIEW

File: openpgp.d.ts [hunk 1]
```diff
@@ -98,6 +98,10 @@ export class Subkey {
   public getCreationTime(): Date;
   public getAlgorithmInfo(): AlgorithmInfo;
   public getKeyID(): KeyID;
+  public getExpirationTime(date?: Date, config?: Config): Promise<Date | typeof Infinity | null>
+  public clone(): Subkey;
```
Comment (larabr) [inline openpgp.d.ts:102]:
```suggestion
```
Not sure about exposing this one. Do you have a use case for it?

Comment (FabienForestier) [reply openpgp.d.ts]: Actually I don't, I'm used to work with typescript and assumed that the method would have been marked private if it wasn't used outside the class. But this is only available in typescript file and not in javascript my bad. I added a commit to remove this line.

File: openpgp.d.ts [hunk 2]
```diff
@@ -98,6 +98,9 @@ export class Subkey {
   public getCreationTime(): Date;
   public getAlgorithmInfo(): AlgorithmInfo;
   public getKeyID(): KeyID;
+  public getExpirationTime(date?: Date, config?: Config): Promise<Date | typeof Infinity | null>
+  public isRevoked(signature: SignaturePacket, key: PublicSubkeyPacket | SecretSubkeyPacket | PublicKeyPacket | SecretKeyPacket, date?: Date, config?: Config): Promise<boolean>;
```
Comment (larabr) [inline openpgp.d.ts:102]:
```suggestion
  public isRevoked(signature: SignaturePacket, key: AnyKeyPacket, date?: Date, config?: Config): Promise<boolean>;
```

**Comment by FabienForestier** [comment]
I added a commit with your suggested changes ;)

**Review by larabr — APPROVED** [review]
Thanks again!

**Pull request merged**